xptifw/src/CMakeLists.txt

```diff
@@ -8,17 +8,22 @@ if (NOT SYCL_EMHASH_DIR)
   include(FetchEmhash)
 endif()
 
-set(PARALLEL_HASHMAP_REPO https://github.com/greg7mdp/parallel-hashmap.git)
-message(STATUS "Will fetch parallel-hashmap from ${PARALLEL_HASHMAP_REPO}")
-FetchContent_Declare(parallel-hashmap
-  GIT_REPOSITORY    ${PARALLEL_HASHMAP_REPO}
-  GIT_TAG           8a889d3699b3c09ade435641fb034427f3fd12b6
-)
-
-FetchContent_GetProperties(parallel-hashmap)
-FetchContent_MakeAvailable(parallel-hashmap)
+find_path(PHMAP_LOC phmap_base.h PATH_SUFFIXES parallel_hashmap)
+if(PHMAP_LOC)
+  set(XPTIFW_PARALLEL_HASHMAP_HEADERS "${PHMAP_LOC}")
```

Comment:
Just thinking out loud, could it happen that the pre-installed parallel hashmap version differs from the one we expect? If so, should we add a check version check here?

Reply:
Thanks, I thought grepping the source to do a version check would be gross but I don't think it will be that bad. Will add a check and print the version.

Reply:
added in latest commit!

Reply:
LGTM, thanks.